target/linux/mvebu/cortexa9_neon/base-files/etc/uci-defaults/03_wireless

```diff
@@ -0,0 +1,47 @@
+#
```

Comment:
shebang line is missing

Reply:
According to @adrianschmutzler uci-defaults do not need it

Reply:
The file is sourced and non-executable, so technically no shebang is required.
This applies to all files in /etc/uci-defaults